README.md

```diff
@@ -13,3 +13,23 @@ The *tutorials* folder contains walkthroughs which demonstrate basic Certifai fu
+| [certifai_multiclass_evaluation_part2.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/multiclass) | Illustrates the second part of the multiclass classification task type. | Multiclass Classification | Logistic Regression | Performance Metric: Accuracy <br /> Explainability-Explanations <br /> Fairness (by feature) | One-hot encoding <br /> Target encoded features |
+| [CertifaiSageMakerExample.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/sagemaker) - Running Cortex Certifai Scan on Sklearn model built and deployed on AWS Sagemaker using Certifai Model Connectors | Uses ml.m4.xlarge Sagemaker notebook instance to create sklearn models to classify german credit loan risk (predict whether loan will be granted or not) and evaluate fairness. | Binary Classification | SVM <br /> Logistic Regression | Fairness | Sagemaker |
+| [CleanStart.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/clean_start) - Building a scan programmatically |  Illustrates the use of Certifai to create and run a fairness scan of a locally defined model from first principles. Again, one of the scan runs produced by this notebook will be saved for viewing |  Binary classification |  SVM <br /> Logistic Regression |  Fairness (by feature) |   |
+| [CleanStart(soft-output).ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/clean_start) - Building a scan programmatically |  A beta version of the CleanStart notebook for models that produce soft outputs (like probabilities) rather than discrete labels. |  Binary classification | SVM <br /> Logistic Regression | Fairness (by feature) | Soft outputs |
+| [CleanStartSegmented.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/segmented_model_demo) - Building a scan programmatically |  Provides an initial illustration of how to accommodate segmented models in Certifai notebooks. |  Binary classification | SVM <br /> Logistic Regression  | Fairness (by feature) |  Segmented models |
+| [FairnessMetrics.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/fairness_metrics) - Using Different Fairness Metrics | Builds a scan definition to perform multiple fairness analyses other than the burden-based default used by the Certifai counterfactual framework. Specifically, it examines two widely used measures of fairness: demographic parity and equal opportunity. | Binary Classification | SVM <br /> Logistic Regression | Fairness | Demographic parity <br /> Equal opportunity <br /> Ground Truth |
```

Comment:
Not sure I really get the distinction of Fairness, Fairness (by feature), Fairness (burden) for evaluation type. They're all by feature, and only one of them deals with fairness other than burden.

Reply:
I removed the parenthetical add ons after fairness